pwiz_tools/Skyline/Skyline.Designer.cs

```diff
@@ -1802,15 +1811,41 @@ private void InitializeComponent()
             this.toolStripSeparator2.Name = "toolStripSeparator2";
             resources.ApplyResources(this.toolStripSeparator2, "toolStripSeparator2");
             // 
+            // searchStripMenuItem
+            // 
+            this.searchStripMenuItem.DropDownItems.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.runPeptideSearchToolStripMenuItem,
+            this.encyclopeDiaSearchMenuItem,
+            this.importFeatureDetectionMenuItem});
+            this.searchStripMenuItem.Name = "searchStripMenuItem";
+            resources.ApplyResources(this.searchStripMenuItem, "searchStripMenuItem");
+            // 
+            // runPeptideSearchToolStripMenuItem
+            // 
+            this.runPeptideSearchToolStripMenuItem.Name = "runPeptideSearchToolStripMenuItem";
+            resources.ApplyResources(this.runPeptideSearchToolStripMenuItem, "runPeptideSearchToolStripMenuItem");
+            this.runPeptideSearchToolStripMenuItem.Click += new System.EventHandler(this.runPeptideSearchToolStripMenuItem_Click);

```

Comment:
The following line needs to be added:
`this.modeUIHandler.SetUIMode(this.runPeptideSearchToolStripMenuItem, pwiz.Skyline.Util.Helpers.ModeUIExtender.MODE_UI_HANDLING_TYPE.proteomic);`
You can achieve this by setting the "UIMode on modeUIHandler" in the property sheet:
![image](https://github.com/ProteoWizard/pwiz/assets/303203/d81ec6f4-b6b9-4e7b-bf8a-4c9835d837c8)

This line is necessary because otherwise if you do "View > User Interface > Molecule" this menu item will change to "Run Molecule Search" when we actually want the menu item to be hidden.

Reply:
Done: https://github.com/ProteoWizard/pwiz/pull/3004/commits/5cfb6e61fb540a5c2c7178c8acd2c9bfe1cc8c0f